xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -243,6 +242,11 @@ void CPVRClient::SetConnectionState(PVR_CONNECTION_STATE state)
     if (!GetAddonProperties())
       CLog::LogF(LOGERROR, "Error reading PVR client properties");
   }
+  else
+  {
+    if (!GetAddonNameStringProperties())
+      CLog::LogF(LOGINFO, "Cannot read PVR client name string properties");
```

Comment:
This should be a LOGERROR, right?

Reply:
I can actually see no use case for setting these strings to return an error.  These wouldn't have been logged before in these state so the info is to the developer.

Reply:
> I can actually see no use case for setting these strings to return an error.

Those strings are obtained via add-on API calls, which can return any error. And in this case `GetAddonNameStringProperties` returns false which should be logged as `LOGERROR` here. `LOGINFO`is for information, but not to make somebody aware of an error.

Reply:
I made the change but I don't consider GetBackendVersion() returning PVR_ERROR_NOT_IMPLEMENTED when it is not connected is an error.  Even when connected it is questionable but I didn't want to change something that causes an add-on about right now.